src/v2/guide/comparison.md

```diff
@@ -1,78 +1,78 @@
 ---
-title: Comparison with Other Frameworks
+title: Perbandingan dengan framework lainnya
 type: guide
 order: 801
 ---
 
-This is definitely the most difficult page in the guide to write, but we do feel it's important. Odds are, you've had problems you tried to solve and you've used another library to solve them. You're here because you want to know if Vue can solve your specific problems better. That's what we hope to answer for you.
+Panduan ini merupakan panduan yang paling susah untuk ditulis, tapi kami rasa sangat penting. Kemungkinannya adalah, kalian memiliki masalah yang kalian coba pecahkan dan kalian menggunakan library lain untuk memecahkannya. Kalian disini karena kalian penasaran apakah Vue dapat memecahkan masalah spesifik dengan cara yang lebih baik. Hal itulah yang kami harapkan untuk menjawab kalian.
 
-We also try very hard to avoid bias. As the core team, we obviously like Vue a lot. There are some problems we think it solves better than anything else out there. If we didn't believe that, we wouldn't be working on it. We do want to be fair and accurate though. Where other libraries offer significant advantages, such as React's vast ecosystem of alternative renderers or Knockout's browser support back to IE6, we try to list these as well.
+Kami juga berusaha sangat keras untuk menghindari penyimpangan. Sebagai tim utama, tentu saja kami sangat menyukai Vue. Kami pikir terdapat beberapa masalah yang diselesaikan secara lebih baik daripada semua hal di luar sana. Jika kami tidak mempercayai hal tersebut, kami tidak akan mengerjakan hal tersebut. Walaupun kami berkeinginan untuk jujur dan akurat. Dimana library lain menawarkan keunggulan tersendiri, contohnya ekosistem _render_ alternative yang luas atau dukungan browser jauh kebelakang hingga IE6, kami juga mencoba untuk mendaftar hal tersebut.
```

Comment:
```suggestion
Kami berusaha sangat keras untuk menghindari kecondongan terhadap Vue. Sebagai tim utama, tentu saja kami sangat menyukai Vue. Kami pikir terdapat beberapa masalah yang diselesaikan secara lebih baik daripada semua hal di luar sana. Jika kami tidak mempercayai hal tersebut, kami tidak akan mengerjakan hal tersebut. Walaupun kami berkeinginan untuk jujur dan akurat. Dimana library lain menawarkan keunggulan tersendiri, contohnya ekosistem _render_ alternative yang luas atau dukungan browser jauh kebelakang hingga IE6, kami juga mencoba untuk mendaftar hal tersebut.
```

Reply:
"Also" in the english version explain that they're avoiding any bias and making this guide helpful simultaneously. With the "Also" removed from the translations, the guide just trying to help the users **then** avoid any bias.

Reply:
I think the "Also" here is to tell they're making the Guide to help users while avoding any bias **simultaneously** instead of avoiding any bias **after** the helping the user with the guide. So the translation "Also" needs to be included as well. 
```suggestion
Kami juga berusaha sangat keras untuk menghindari kecondongan terhadap Vue. Sebagai tim utama, tentu saja kami sangat menyukai Vue. Kami pikir terdapat beberapa masalah yang diselesaikan secara lebih baik daripada semua hal di luar sana. Jika kami tidak mempercayai hal tersebut, kami tidak akan mengerjakan hal tersebut. Walaupun kami berkeinginan untuk jujur dan akurat. Dimana library lain menawarkan keunggulan tersendiri, contohnya ekosistem _render_ alternative yang luas atau dukungan browser jauh kebelakang hingga IE6, kami juga mencoba untuk mendaftar hal tersebut.
```